lib/parsing.js

```diff
@@ -25,23 +25,73 @@
  * @property {AstNode[]} [params]
  */
 
-const {SyntaxError: JSDocTypeSyntaxError, parse} = require('../peg_lib/jsdoctype.js');
+const {
+  parse, SyntaxError: JSDocTypeSyntaxError,
+} = require('../peg_lib/jsdoctype-permissive.js');
+
+const {
+  parse: parseJsdoc, SyntaxError: JSDocSyntaxError,
+} = require('../peg_lib/jsdoctype-jsdoc.js');
+const {
+  parse: parseClosure, SyntaxError: ClosureSyntaxError,
+} = require('../peg_lib/jsdoctype-closure.js');
+const {
+  parse: parseTypeScript, SyntaxError: TypeScriptSyntaxError,
+} = require('../peg_lib/jsdoctype-typescript.js');
 
 module.exports = {
 
-  /** * A class for JSDoc type expression syntax errors.
+  /**
+   * A class for JSDoc type expression syntax errors.
    * @constructor
    * @extends {Error}
    */
   JSDocTypeSyntaxError,
 
+  /**
+   * A class for JSDoc type expression syntax errors.
```

Comment:
descriptions should probably be different for each

Reply:
I've added this for now. The PEG comments are converted into different JS files through the `npm` `build` script additions in `package.json`.